contracts/test/foundry/FleekERC721/AccessControl.t.sol

```diff
@@ -425,4 +425,68 @@ contract Test_FleekERC721_AccessControl is Test_FleekERC721_Base {
         vm.prank(tokenOwner);
         CuT.burn(tokenId);
     }
+
```

Comment:
Should this go into another test file? Just wondering if access control is the best place or might be hard to tell that from the name. Maybe the pausable file or main erc721 test file.

Reply:
Here it is just testing if the address is able or not to call the function and not the function run itself, the same as other functions in this test suit.